lib/build.gradle

```diff
@@ -302,7 +314,7 @@ task checkCurrentJavaVersion() {
 
 task generateHeader(type: Exec) {
     workingDir "${projectDir}/src/main/java/org/kiwix/"
-    commandLine 'bash', '-c', "javac -h ${buildDir}/include/javah_generated/ -d ${projectDir}/src/test/ ${getLibzimFiles()} ${getLibkiwixFiles()}"
+    commandLine 'bash', '-c', "javac -h ${buildDir}/include/javah_generated/ -d ${projectDir}/src/test/java/ ${getLibzimFiles()} ${getLibkiwixFiles()}"
```

Comment:
@mgautierfr , we are generating java subdirectory in test via ```buildHeaders```, Because we are generating the test source jar from .class files, which we are using in our test cases.

Reply:
Ok.
It would be better to not generate things in source directory. Everything should be generated in the build directory.

Reply:
Since we are compile ```test.java``` file we required java sources in the same directory , I tried from build but its importing.

Reply:
When you are building the kiwix-android application, you don't need the java sources of the wrapper.
Why do you need the source of the wrapper when you build the test application ?